frontend/server/src/main/java/org/pytorch/serve/util/NettyUtils.java

```diff
@@ -240,7 +240,7 @@ public static InputParameter getFormData(InterfaceHttpData data) {
             case Attribute:
                 Attribute attribute = (Attribute) data;
                 try {
-                    return new InputParameter(name, attribute.getValue());
+                    return new InputParameter(name, getBytes(attribute.getByteBuf()));
```

Comment:
@jonathan-conder-sm: Please add a test case in `ModelServerTest.java` and also provide a detailed example in the PR comment about how this fix works.

Reply:
I added some tests and will edit the comment shortly